oci.go

```diff
@@ -245,8 +260,15 @@ func processCgroupsPathForResource(ociSpec oci.CompatOCISpec, resource string) (
 		}
 	}
 
-	if cgroupMountFound == false {
-		return "", fmt.Errorf("cgroupsPath is absolute, cgroup mount MUST exist")
+	if !cgroupMountFound {
+		if isPod {
+			return "", fmt.Errorf("cgroupsPath %q is absolute, cgroup mount MUST exist",
```

Comment:
I don't think we should modify this error log. This log explains what the OCI spec expects from an absolute path. And this patch does not modify this behavior, it only makes a particular case for CRI-O because there is a bug, they are not handling this OCI specification here. This patch is kind of a workaround to get CRI-O working without modifying the behavior for Docker.